dist/tools/flatc/Makefile

```diff
@@ -0,0 +1,29 @@
+PKG_NAME=flatc
+PKG_URL=https://github.com/google/flatbuffers
+PKG_VERSION=9e7e8cbe9f675123dd41b7c62868acad39188cae
+PKG_LICENSE=Apache2.0
+
+PKG_BUILDDIR=$(CURDIR)/bin
+
+# manually set some RIOT env vars, so this Makefile can be called stand-alone
+RIOTBASE ?= $(CURDIR)/../../..
+RIOTTOOLS ?= $(CURDIR)/..
+GITCACHE ?= $(RIOTTOOLS)/git/git-cache
+
+CMAKE_OPTIONS = \
+    -DCMAKE_BUILD_TYPE=Release       \
+    -DFLATBUFFERS_BUILD_TESTS=OFF    \
+    -DFLATBUFFERS_BUILD_FLATHASH=OFF \
+    -DFLATBUFFERS_BUILD_FLATLIB=OFF  \
+    #
+
+include $(RIOTBASE)/pkg/pkg.mk
+
+all:
+	@echo "[INFO] compiling flatc from source now"
+	@cd "$(PKG_BUILDDIR)" && env -i PATH="$(PATH)" cmake -G "Unix Makefiles" $(CMAKE_OPTIONS) .
+	@env -i PATH="$(PATH)" "$(MAKE)" -C "$(PKG_BUILDDIR)"
```

Comment:
I don't think "PATH" needs to be based here.

Reply:
It does otherwise cmake and related configuration files are not found.

Reply:
For `cmake` yes, but for `MAKE` it wasn't needed for it to compile on my machine.

Reply:
ah, yes! you are right this is not needed for the make command... => removed